activity_calendar/models.py

```diff
@@ -259,6 +263,14 @@ def __new__(mcs, name, bases, attrs, **kwargs):
                         new_field.name = new_field_name
 
                         attrs[new_field_name] = new_field
+
+                        if isinstance(field, models.BooleanField):
+                            # Override choices for BooleanField with null=True.
+                            # Without this, the null-selection shows up as 'unknown' (which is misleading)
+                            new_field.choices = BLANK_CHOICE_DASH + [
+                                (True,  _('Yes')),
+                                (False, _('No')),
+                            ]
                         break
```

Comment:
I personally do not like this implementation, though I will except it, it would make much more create a widget for this and set this in the form instead (https://docs.djangoproject.com/en/3.1/ref/forms/widgets/). Note that widgets have a method `value_from_datadict(self, data, files, name)` that is called by the Form to translate the datadict (POST or otherwise) to the relevant.
Edit: There is a `NullBooleanSelect` widget that has an empty, yes and no option that it displays. This might be a lot better.

Reply:
The NullBooleanSelect would actually be used without this snippet. However, this resulted in "Unknown" being the option for a null-value without any way to override this text.

https://github.com/django/django/blob/adb40d217ec57ade46b1394cfbf3c513dc669445/django/forms/widgets.py#L708-#L718

I do agree that a custom widget results in cleaner code.

Reply:
Ik zie twee opties:
1) Unknown word in de widget eerst door een gettext_lazy gegooid, dus je kan het in de vertaling overschrijven
2) Subclass NullBooleanSelect en in de init zet zelf self.choices in de init method (je kan een inheritence stap overslaan door `super(Select).__init__(attrs, choices)` of je kan na de super aan te roepen self.choices = [('unknown', 'Laat leeg'), ('true', 'Ja'), ('false', 'Nee') ]